lib/AST/ClangTypeConverter.cpp

```diff
@@ -158,6 +158,57 @@ const clang::Type *ClangTypeConverter::getFunctionType(
   llvm_unreachable("invalid representation");
 }
 
+const clang::Type *ClangTypeConverter::getFunctionType(
```

Comment:
Since SILFunctionType::get() and friends live in lib/SIL, maybe this should go there as well?

Reply:
Leaving this as-is based on offline discussion.